DiceKitTests/Die_Tests.swift

```diff
@@ -268,55 +247,50 @@ extension Die_Tests {
     
     func common_RollerType_shouldReturnWithinPositiveRangeForSidesGreaterThan1(rollerType: Die.RollerType) {
         property("generates values within range of 1...sides") <- forAll {
-            (i: PositiveSidesType) in
-            
-            guard i > 1 else { return true }
-            
-            let sides = Int(i)
-            
-            let result = rollerType(sides: sides)
+            (i: Int) in
             
-            return result > 0 && result <= sides
+            return (i > 1) ==> {
+                let result = rollerType(sides: i)
+                
+                return result > 0 && result <= i
+            }
         }
     }
     
     func common_RollerType_shouldReturnWithinNegativeRangeForSidesLessThan0(rollerType: Die.RollerType) {
         property("generates values within range of sides..<0") <- forAll {
-            (i: PositiveSidesType) in
-            
-            guard i > 1 else { return true }
-            
-            let sides = -Int(i)
-            
-            let result = rollerType(sides: sides)
+            (i: Int) in
             
-            return result < 0 && result >= sides
+            return (i < -1) ==> {
+                let result = rollerType(sides: i)
+                
+                return result < 0 && result >= i
+            }
         }
     }
     
     func common_RollerType_shouldReturnWithinRangeForSides(rollerType: Die.RollerType) {
         property("generates values within range of -sides...sides") <- forAll {
-            (i: PositiveSidesType) in
-            
-            let sides = Int(i)
-            
-            let result = rollerType(sides: sides)
+            (i: Int) in
             
-            return (-sides...sides).contains(result)
+            // Int.min has one more value than Int.max, making this crash when negating it
```

Comment:
Though, I think that has a chance of doing fewer checks, because of duplication. I think I'll leave it as is.